catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -263,15 +273,25 @@ public void configure() throws Exception {
                 // Configure the camel route to ignore changing file (larger files that are in the process of being copied)
                 // Set the readLockTimeout to continuously poll the directory so long as the directory monitor exists
                 // Set the readLockCheckInterval to check every 5 seconds
-                String inbox = "file:" + monitoredDirectory + "?moveFailed=.errors&readLock=changed&readLockTimeout=0&readLockCheckInterval=5000";
-                if (copyIngestedFiles) {
-                    inbox += "&move=.ingested";
-                } else {
-                    inbox += "&delete=true";
+                StringBuilder inbox = new StringBuilder("file:" + monitoredDirectory
+                        + "?recursive=true&moveFailed=.errors&readLock=changed&readLockTimeout=0&readLockCheckInterval=5000");
```

Comment:
Yes, this has been requested for awhile and happened to be so easy to fix that I added it while I was in the area.